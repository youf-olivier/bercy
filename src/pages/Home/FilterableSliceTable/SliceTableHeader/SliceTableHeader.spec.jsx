import { render } from "@testing-library/react";
import { SliceTableHeader } from "pages/Home/FilterableSliceTable/SliceTableHeader/SliceTableHeader";

describe("Should render Table Header", () => {
  it("SnapShot testing", () => {
    const { asFragment } = render(
      <table>
        <SliceTableHeader />
      </table>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
