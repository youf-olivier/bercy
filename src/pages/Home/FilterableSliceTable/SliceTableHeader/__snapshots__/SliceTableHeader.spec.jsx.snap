// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Table Header SnapShot testing 1`] = `
<DocumentFragment>
  <table>
    <thead
      class="af-table-thead"
    >
      <tr
        class="af-table__tr"
      >
        <th
          class="af-table__th"
        >
          <span
            class="af-table-th-content"
          >
            Tranches
          </span>
        </th>
        <th
          class="af-table__th"
        >
          <span
            class="af-table-th-content"
          >
            Taux d'imposition
          </span>
        </th>
      </tr>
    </thead>
  </table>
</DocumentFragment>
`;
